### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by Winstonjames2** [GreaterWMS/GreaterWMS#359] [General]
what does frontend do, i don't really know what does it do, i am beginner so i don't know much about quasar. yes i am still figuring out. i mean like localhost:8008 and localhost:8080 are provide the same right?

**Comment by Singosgu** [comment]
yup

**Comment by Winstonjames2** [comment]
I am still wondering what does http://localhost:8080/ this address used for.

**Reply by Singosgu** [reply]
just for develop

**Comment by Winstonjames2** [comment]
thanks i am fully understand how this project works. At first I'm not familiar with quasar framework. Thanks again

**Comment by Winstonjames2** [comment]
I am wondering if it is okay to use node 14 version and some outdated and legacy version. I tried to set high version, but some is not supported by vue3, etc. What should I do. I love this, out of the box system, like this.

**Reply by Singosgu** [reply]
GreaterWMS 3.0 will fully support vue3

**Reply by Winstonjames2** [reply]
thanks can't wait
